server/matcher/match.go

```diff
@@ -151,6 +153,23 @@ func (m *Matcher) Match(book Booker, queue []*OrderRevealed) (seed []byte, match
 	// entries.
 	partialMap := make(map[order.OrderID]*order.LimitOrder)
 
+	// Track initially unmatched standing limit orders and remove them if they
+	// are matched down-queue so that they aren't added to the nomatched slice.
```

Comment:
Noticing that this still refers to unmatched _standing_ limit orders, but it now covers all limits.